docs/changelog.rst

```diff
@@ -2,6 +2,16 @@
 Changelog
 =========
 
+* :feature:`-` Added support for the following tokens:
```

Comment:
The only thing is this should be at the end of the changelog for the version but looks fine.